src/main/java/Parser.java

```diff
@@ -0,0 +1,120 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The Parser class deals with making sense of the user's command.
+ */
+public class Parser {
+    /**
+     * Gets a Command object
+     *
+     * @param command A string representing the user's command.
+     * @return A Command object.
+     */
+    public static Command parse(String command) {
+        return new Command(command);
+    }
+
+    /**
+     * Gets the string representation of a date by converting them from dd/mm/yyyy hhmm
+     * to english sentences.
+     *
+     * @param time A string representing date in the form of dd/mm/yyyy hhmm
+     * @return A string representation of a date in english sentences.
+     */
+    public static String convertDateAndTime(String time) { // in the form of dd/mm/yyyy hhmm
```

Comment:
You could use java's inbuilt java.time (aka JodaTime) library to do the parsing ( and even formatting output ) instead!
https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html
Classes relavant ( I believe ): DateTimeFormatter, TimeFormatter, LocalDateTime, LocalTime

Regarding the java documentation (while browsing the docs):
There are many date time related classes outside of the java.time which are deprecated and harder to use (pre java8 apis).